test/larger_results_sidecar_logs_test.go

```diff
@@ -114,6 +124,12 @@ func TestLargerResultsSidecarLogs(t *testing.T) {
 					t.Fatalf(`The expected taskrun does not match created taskrun. Here is the diff: %v`, d)
 				}
 			}
+
+			// reset result extraction method feature flag
+			if err := updateConfigMap(ctx, c.KubeClient, system.Namespace(), config.GetFeatureFlagsConfigName(), map[string]string{"results-from": previousResultExtractionMethod}); err != nil {
+				t.Fatal(err)
+			}
+
```

Comment:
Should we wrap this in a function and we can defer it? That way, if there is an error, the config will be reset so that it does not interfere with tests later.

Reply:
Good point! Done. PTAL!